windows/utils/win_install_online_updates.yml

```diff
@@ -16,13 +16,25 @@
 #   win_updates_log_file (optional): The log file path for Windows Update in guest OS.
 #     Default is 'C:\win_updates_log.txt'.
 # Return:
-#   win_online_updates_installed: Indicates if there are Windows online updates
-#     installed and the installed count is the same as the found count.
+#   win_online_updates_succeed: Will be set to 'true' in below conditions:
+#     1. no Windows updates found,
+#     2. there are Windows updates found, and the installed updates count is equal to
+#        the found count,
+#     3. there are Windows updates found, and the installed updates count is not equal
+#        to the found count, but the failure messages of the not installed ones contain
+#        the keywords representing failure can be ignored.
+#        For example, 'failure_msg' is "Operation was not performed because there are
+#        no applicable updates (WU_E_NOT_APPLICABLE 0x80240017)".
+#     Other cases, it will be set to 'false'.
+#   win_online_updates_installed: If set to 'true', there are Windows online updates
+#     installed in guest OS, or it will be 'false'.
```

Comment:
Maybe it's more clear. 
"there are Windows online updates installed in guest OS" ---> "there are **at least one** Windows online updates installed in guest OS".

Reply:
Updated, thanks.